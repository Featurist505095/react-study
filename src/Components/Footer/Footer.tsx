import React, { FunctionComponent } from "react";
import FooterStyled from "./Footer.styled";
import CompanyName from "../CompanyName";

const Footer: FunctionComponent = () => {
  return (
    <FooterStyled>
      <CompanyName />
    </FooterStyled>
  );
};

export default Footer;