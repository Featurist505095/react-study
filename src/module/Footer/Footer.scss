footer {
    padding: 15px 40px;
    background: dimgray;
}