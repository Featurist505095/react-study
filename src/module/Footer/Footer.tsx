import React, { FunctionComponent } from "react";
import { CompanyName } from "../CompanyName/CompanyName";
import "./Footer.scss";

export const Footer: FunctionComponent = () => {
  return (
    <footer>
      <CompanyName />
    </footer>
  );
};