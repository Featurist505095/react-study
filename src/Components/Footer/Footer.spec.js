import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

it('Test footer render', () => {
    const tree = renderer
        .create(<Footer />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});