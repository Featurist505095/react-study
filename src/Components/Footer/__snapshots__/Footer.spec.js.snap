// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="sc-bdfBwQ gVCbTU"
>
  <a
    className="company-name"
    href="/search"
  >
    <h3>
      netflixroulette
    </h3>
  </a>
</footer>
`;
